tests/test_utils.cpp

```diff
@@ -42,9 +43,9 @@ std::string fileContents(const std::string &fileName)
 
 void printIssues(const libcellml::LoggerPtr &l, bool headings, bool causes, bool rule)
 {
+    int width = int(floor(log10(l->errorCount())));
```

Comment:
Wowee ... this is scary ... we're expecting more than 999 errors!! 😮

Reply:
Maybe not, but I don't hard-coded solutions, so... :)